### PR TITLE
Issue #478

### PR DESCRIPTION
**Pull request by kirjaamo**
Modes added to wsrep_mode

### CODE REVIEW

File: galeracluster/source/documentation/mysql-wsrep-options.rst [hunk 1]
```diff
@@ -1005,19 +1005,39 @@ Extends node behaviour with provided values.
...
 .. csv-table::
    :class: doc-options
-   :header: "Value", "Behaviour"
-   :widths: 25, 50
+   :header: "Galera Cluster version", "MySQL database server version", "Blog post"
+
+The options for MySQL are:
+
+- ``IGNORE_NATIVE_REPLICATION_FILTER_RULES`` - Ignore native replication filter rules for cluster events. In other words, native asynchronous replication filtering options are honored when applying Galera replication. These options are of format ``replicate_*``, and specify if transactions for a table or a database should be applied or not.
+- ``IGNORE_CASCADING_FK_DELETE_MISSING_ROW_ERROR`` - Ignore missing row errors when applying a cascading delete write set. This a workaround for https://bugs.mysql.com/bug.php?id=80821, and is possibly obsolete in the upstream versions.
```
Comment (byte) [inline galeracluster/source/documentation/mysql-wsrep-options.rst:1020]: can you make this an actual link?

File: galeracluster/source/documentation/mariadb-options.rst [hunk 2]
```diff
@@ -229,95 +227,6 @@ Wsrep patch version, for example ``wsrep_25.10``.
...
-    | Variable_name                | Value |
-    +------------------------------+-------+
-    | wsrep_mode_replicate_myisam  | ON    |
-    +------------------------------+-------+
-
-
-
 .. _`mariadb_enterprise_options`:
 .. rst-class:: section-heading
 .. rubric:: MariaDB Enterprise Options
```
Comment (byte) [inline galeracluster/source/documentation/mariadb-options.rst:232]: are there MariaDB Enterprise options?

**Review by byte — CHANGES_REQUESTED** [review]
as per review

File: galeracluster/source/documentation/mysql-wsrep-options.rst (hunk 1 above)
Comment (kirjaamo) [reply galeracluster/source/documentation/mysql-wsrep-options.rst:1020]: In the conversion that I am looking at, it already is a link. Does it not work on your conversion?

File: galeracluster/source/documentation/mariadb-options.rst (hunk 2 above)
Comment (kirjaamo) [reply galeracluster/source/documentation/mariadb-options.rst:232]: Yes:

image

File: galeracluster/source/documentation/mysql-wsrep-options.rst (hunk 1 above)
Comment (byte) [reply galeracluster/source/documentation/mysql-wsrep-options.rst:1020]: Hmm, ok, i think it does now. Weird

**Review by byte — APPROVED** [review]
ok

**Pull request merged**